app/controllers/attendances_controller.rb

```diff
@@ -15,12 +15,21 @@ def index
   def create
     @employees_todays_attendance = current_employee.todays_attendance_of_employee
     if @employees_todays_attendance.blank?
+      setting = Setting.all.first
```

Comment:
shouldn't it be something like current_employee.company.settings or current_company.settings

Reply:
already solved it in my current branch